go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -40,11 +40,10 @@ import (
 // DefaultNetworkController structure is the object which holds the controls for starting
 // and reacting upon the watched resources (e.g. pods, endpoints) for default l3 network
 type DefaultNetworkController struct {
-	BaseNetworkController
+	NetworkControllerInfo
```

Comment:
would a better name be CommonNetworkController? So then we have:

BaseNetworkController->CommonNetworkController->DefaultNetworkController
                                                                                                ->SecondaryNetworkController

wdyt?

Reply:
@jcaamano I agree with you.